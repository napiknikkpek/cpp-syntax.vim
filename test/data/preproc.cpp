#pragma once

#include <map>
#include <boost/hana.hpp>

#define HELLO hello
HELLO
