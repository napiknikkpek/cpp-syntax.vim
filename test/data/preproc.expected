mmmmmmm.mmmm

mmmmmmmm.sssss
mmmmmmmm.ssssssssssssssss

mmmmmmm.mmmmm.mmmmm
mmmmm
